src/main/java/com/example/bookstore/dto/request/BookRequestDto.java

```diff
@@ -1,13 +1,21 @@
 package com.example.bookstore.dto.request;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotBlank
     private String title;
```

Comment:
It's also recommended to validate maximum length of text fields in order to avoid exceptions related to maximum column size in your DB. Use `Size` for this.